tests/snmp/test_snmp_loopback.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+import time
+import logging
+import ipaddress
+try:  # python3 
+    from shlex import quote
+except ImportError:  #  python2
+    from pipes import quote
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.device_type('vs')
```

Comment:
does it mean it will run only on vs ?

Reply:
This is marked to run on vs test, but when running tests, if we specify the name of the test file, that specific test case will be executed on physical DUT or on vs, and marker will not be used. If pytest is triggered to execute all vs marked tests, then this test will be a part of that. Added this as other SNMP test cases have this marker.